ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/ropsten.yaml

```diff
@@ -27,8 +27,11 @@ TERMINAL_TOTAL_DIFFICULTY: 50000000000000000
 TERMINAL_BLOCK_HASH: 0x0000000000000000000000000000000000000000000000000000000000000000
 TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH: 18446744073709551615
 
+# Capella
+CAPELLA_FORK_VERSION: 0x03000071
```

Comment:
there's not really a value that I can use here?

Reply:
Shouldn't need to specify anything.  None of the network yaml files should change at this point - only when we actually need to run capella on them. Otherwise we'd break backwards compatibility.